dpbench/infrastructure/framework.py

```diff
@@ -72,6 +72,10 @@ def device_filter_string(self) -> str:
 
     def version(self) -> str:
         """Returns the framework version."""
+        if self.fname == "python":
+            import platform
+
+            return platform.python_version()
```

Comment:
So this is also the runtime version, not the version we used for building, right?

Reply:
Yes, that is correct.